File: shoring/blackjackEngine/src/test/java/com/panopset/tests/blackjackEngine/DeckPileTest.java
package com.panopset.tests.blackjackEngine;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

import com.panopset.blackjackEngine.DeckPile;

public class DeckPileTest {

  @Test
  void test() {
    DeckPile.reset();
    Assertions.assertTrue(DeckPile.pull());
    Assertions.assertFalse(DeckPile.pull());
    Assertions.assertTrue(DeckPile.pull());
  }
}

File: code/shoring/blackjackEngine/src/test/kotlin/com/panopset/tests/blackjackEngine/DeckPileTest.kt
package com.panopset.tests.blackjackEngine

import com.panopset.blackjackEngine.DeckPile.pull
import com.panopset.blackjackEngine.DeckPile.reset
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class DeckPileTest {
    @Test
    fun test() {
        reset()
        Assertions.assertTrue(pull())
        Assertions.assertFalse(pull())
        Assertions.assertTrue(pull())
    }
}
